po/Makevars

```diff
@@ -0,0 +1,82 @@
+# Makefile variables for PO directory in any package using GNU gettext.
+#
+# Copyright (C) 2003-2019 Free Software Foundation, Inc.
+# This file is free software; the Free Software Foundation gives
+# unlimited permission to use, copy, distribute, and modify it.
+
+# Usually the message domain is the same as the package name.
+DOMAIN = $(PACKAGE)
+
+# These two variables depend on the location of this directory.
+subdir = po
+top_builddir = ..
+
+# These options get passed to xgettext.
+XGETTEXT_OPTIONS = --keyword=_ --keyword=N_ --keyword=C_:1c,2 --keyword=NC_:1c,2 --from-code=UTF-8 -c"Translators:"
```

Comment:
Beware: as we changed this since last re-generation, running update-po might loos the `#|` entries.  We might then want to revert the PO files to their previous state, and only *then* run update-po to get as much info as possible.  Not sure if the loosing is expected though, but that's what I noticed.

Reply:
If I understand you correctly, we could drop the commit e86ae47a64ac257ac09d98b2ad81a335b4cf9f1f and regenerate the files again?

Reply:
Yes that's my idea